File: src/main/java/org/bql/rooms/dice/model/DiceDto.java
package org.bql.rooms.dice.model;

import org.bql.utils.builder_clazz.ann.Protostuff;

@Protostuff
public class DiceDto {
    private int one;
    private int two;
    /**场次*/
    private long battleCount;
    public DiceDto() {
    }

    public DiceDto(int one, int two, long battleCount) {
        this.one = one;
        this.two = two;
        this.battleCount = battleCount;
    }

    public int getOne() {
        return one;
    }

    public void setOne(int one) {
        this.one = one;
    }

    public int getTwo() {
        return two;
    }

    public long getBattleCount() {
        return battleCount;
    }

    public void setBattleCount(long battleCount) {
        this.battleCount = battleCount;
    }

    public void setTwo(int two) {
        this.two = two;
    }
}

File: cs/DiceDto.cs
using System;
using System.Collections.Generic;
using ProtoBuf;
[ProtoContract]
public class DiceDto : IProtostuff {
	[ProtoMember(1)]
	public int One{get;set;}
	[ProtoMember(2)]
	public int Two{get;set;}
	[ProtoMember(3)]
	public long BattleCount{get;set;}
}
